tests/test_core.py

```diff
@@ -8661,7 +8695,7 @@ def test_em_async_js(self):
 
 
 # Generate tests for everything
-def make_run(name, emcc_args, settings=None, env=None):
+def make_run(name, emcc_args, settings=None, env=None, wasm64=False):
```

Comment:
Maybe make this `node_args=None` so `--experimental-wasm-bigint` can be specified down on line 8753